openslides/users/static/js/users/site.js

```diff
@@ -458,6 +475,62 @@ angular.module('OpenSlidesApp.users.site', [
             }
             $scope.sortColumn = column;
         };
+        // define custom search filter string
+        $scope.getFilterString = function (user) {
+            return [
+                user.first_name,
+                user.last_name,
+                user.title,
+                user.number,
+                user.comment,
+                user.structure_level,
+                _.map(
+                    user.groups,
+                    function (group) {
+                        return group.name;
+                    }
+                ).join(" "),
+            ].join(" ");
+        };
+        $scope.multiselectFilter = {
+            group: [],
+        };
+        $scope.getItemId = {
```

Comment:
Here the same. If we have a new style filter table service, it can be configured with single and multi select filters and provide this code in a generic way.